Microsoft.Packaging.Tools/tasks/targets/Microsoft.Packaging.Tools.targets

```diff
@@ -61,6 +61,7 @@
                                 ReferenceCopyLocalPaths="@(ReferenceCopyLocalPaths)"
                                 OtherRuntimeItems="@(_LockFileAssemblies)"
                                 PlatformManifests="@(PackageConflictPlatformManifests)"
+                                ForcedPackages="$(PackageConflictForcedPackages)"
```

Comment:
Given that you are just taking a set of PackageId's someone could get into a state where they want to prefer one package for one binary and another package for another binary and if those packages have multiple binaries in them then it might be difficult. I don't think I would worry about it much right now but in the future we might want to consider this being an item group with metadata, perhaps we do that now to make such a transition easier.